extensions/expanded-registers/README.md

```diff
@@ -7,31 +7,14 @@
 
 # Overview
 
-This extension adds an instruction prefix byte which expands the number of available registers from 8 registers to 32 registers. The aliases below along with who saves it is for if the stack and functions extension is implemented
-
-| Register | Alias | Who saves it |
-|----------|-------|--------------|
-| `r8`     | `t0`  | Caller Saved |
-| `r9`     | `t1`  | Caller Saved |
-| `r10`    | `t2`  | Caller Saved |
-| `r11`    | `t3`  | Caller Saved |
-| `r12`    | `t4`  | Caller Saved |
-| `r13`    | `s2`  | Caller Saved |
-| `r14`    | `s3`  | Caller Saved |
-| `r15`    | `s4`  | Caller Saved |
-
-- aN registers are argument registers and store the first N arguments to a function call. Additional arguments should be pushed to the stack.
-- sN registers are registers that must be saved before re-using.
-- tN registers are temporary registers which a function can use however it wants.
-
-Remember that the above are just suggestions. A complete specification of a standard calling convention for ETC-A is not yet specified.
+This extension adds an instruction prefix byte expanding the number of available registers from 8 registers to 16, and suggests an extension of the [Stack & Functions](../stack-and-functions/readme.md) extension's calling conventions to the added registers.
 
 # Added Instructions
 
 The following 1-byte instruction prefix is now defined.
 
-| `REX`   | | | | |
-|:-------------|---|----|----|----|
+| `REX`  |     |     |     |     |
+|:-------|-----|-----|-----|-----|
```

Comment:
Cleaning these up is fine, but remember that markdown doesn't actually care about the spacing and spacing things "correctly" gets messed up if the table is ever modified. This has come up in our tables before.